zeebe/clients/java/src/main/java/io/camunda/zeebe/client/api/command/UpdateUserTaskCommandStep1.java

```diff
@@ -0,0 +1,117 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.command;
+
+import io.camunda.zeebe.client.api.response.UpdateUserTaskResponse;
+import java.util.List;
+
+public interface UpdateUserTaskCommandStep1 extends FinalCommandStep<UpdateUserTaskResponse> {
+
+  /**
+   * Set the custom action to update the user task with.
+   *
+   * @param action the action value
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 action(String action);
+
+  /**
+   * Set the due date to set in the user task.
+   *
+   * @param dueDate the due date to set
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 dueDate(String dueDate);
+
+  /**
+   * Clear the due date in the user task.
+   *
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 clearDueDate();
+
+  /**
+   * Set the follow-up date to set in the user task.
+   *
+   * @param followUpDate the follow-up date to set
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 followUpDate(String followUpDate);
+
+  /**
+   * Clear the follow-up date in the user task.
+   *
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 clearFollowUpDate();
+
+  /**
+   * Set the candidate groups to set in the user task.
+   *
+   * @param candidateGroups the candidate groups to set
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 candidateGroups(List<String> candidateGroups);
+
+  /**
+   * Set the candidate groups to set in the user task.
+   *
+   * @param candidateGroups the candidate groups to set
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 candidateGroups(String... candidateGroups);
+
+  /**
+   * Clear the candidate groups in the user task.
+   *
+   * @return the builder for this command. Call {@link #send()} to complete the command and send it
+   *     to the broker.
+   */
+  UpdateUserTaskCommandStep1 clearCandidateGroups();
+
+  /**
+   * Set the candidate users to set in the user task.
```

Comment:
:question: Similar to the comment above, should we note that the command will completely override the existing candidate users?

I guess at this point, it's not even possible to get the existing candidate users since we don't have a `UserTask` entity in the client.

Reply:
Same as above 🙂